### PR TITLE
Add Fragnova mainnet and testnet

### PR DESCRIPTION
**Pull request by sinkingsugar**
As announced in our latest press release https://www.pocketgamer.biz/news/80988/fragnova-unveils-new-game-and-user-generated-content-creation-system/
We launched our mainnet.
It would be great to have it added to apps!
Thanks!🙏🙏🙏

### CODE REVIEW

**Review by jacogr — CHANGES_REQUESTED** [review]
Could you just merge in master (based on commits, this means: again...), GH shows conflicts.

**Comment by sinkingsugar** [comment]
Sure 👌 @aang114 can you have a look please?

**Comment by aang114** [comment]
> Sure 👌 @aang114 can you have a look please?

Done. Just merged the latest `master` branch into our branch.

**Comment by sinkingsugar** [comment]
it's done @jacogr 🙏🙏

File: scripts/fixupModules.cjs [hunk 1]
```diff
@@ -7,6 +7,7 @@ const process = require('node:process');
 
 // incorrect packages without type specifier
 [
+  '@fragnova/api-augment',
```
Comment (jacogr) [inline scripts/fixupModules.cjs:10]: Is there an actual issue logged to fix this? If so please do add it like the one below.

(These will be removed within the next 7-ish days, we don't want to continue working around packages that don't quite specify export maps correctly. It is terrible having to fixup as part of the build.)

Comment (aang114) [reply scripts/fixupModules.cjs:11]: @jacogr Sure, I have created the issue and updated this file with the issue URL. Thanks

**Review by jacogr — APPROVED** [review]
Thank you.

(As mentioned, just bear in mind that the current plan is to drop all the fixup stuff in around 1+ week (depending on time availability) - obviously will ensure that all related deps are updated to latest at that point)

**Pull request merged**